internal/provider/provider.go

```diff
@@ -63,7 +63,14 @@ func (p *ProviderFramework) Schema(ctx context.Context, req provider.SchemaReque
 }
 
 func (p *ProviderFramework) Resources(ctx context.Context) []func() resource.Resource {
-	return []func() resource.Resource{}
+	return []func() resource.Resource{
+		func() resource.Resource {
+			return &resourceVaultsecretsApp{}
+		},
+		func() resource.Resource {
+			return &resourceVaultsecretsSecret{}
+		},
+	}
```

Comment:
nit: similar to the comment below, this can just be the list of functions defined in each resource file:

```go
return []func() resource.Resource{
		NewVaultSecretsAppResource,
		NewVaultSecretsSecretResource,
	}
```

You can use any of the utility providers as an example:
- https://github.com/hashicorp/terraform-provider-dns/blob/main/internal/provider/resource_dns_cname_record.go#L29
- https://github.com/hashicorp/terraform-provider-dns/blob/b89204a34cba26de70ecbbe0864283ff17082964/internal/provider/provider_framework.go#L334-L343

Reply:
Good call!